test/e2e/suite/issuers/acme/certificate/dns01.go

```diff
@@ -58,6 +59,9 @@ func testRFC2136DNSProvider() bool {
 
 		dnsDomain := ""
 
+		unsupportedFeatures := featureset.NewFeatureSet(featureset.SaveCAToSecretFeature)
```

Comment:
Perhaps add a comment here explicitly detailing why we are excluding this feature.

Reply:
Done with a link to an issue which, incidentally, has been closed, but looks like it ought to be re-opened for further discussion: https://github.com/jetstack/cert-manager/issues/1571